packages/api/README.md

```diff
@@ -24,7 +24,7 @@ Using `api` is as simple as supplying it an OpenAPI and using the SDK as you wou
 ```js
 const sdk = require('api')('https://raw.githubusercontent.com/readmeio/oas/master/packages/examples/3.0/json/petstore.json');
 
-sdk.listPets().then(res => res.json()).then(res => {
+sdk.listPets().then(res => {
```

Comment:
Starting to wonder if we should update `res` in our code snippets to be `json` so it's clearer that you're getting JSON back and not a `Response` object.